finality-aleph/src/data_io.rs

```diff
@@ -277,17 +288,17 @@ pub(crate) async fn refresh_best_chain<B, BE, SC, C>(
                 } else {
                     // we check if prev_best_header is an ancestor of new_best_header:
                     if data.lock().number < max_block_num {
-                        let reduced_header = reduce_to_num(new_best_header.clone(), max_block_num);
+                        let reduced_header = reduce_header_to_num::<B, BE, C>(client.clone(), new_best_header.clone(), max_block_num);
```

Comment:
Are the template args necessary here? I would have thought `client` contains all the information the function needs.

Reply:
Cool, it works without the explicit args, thanks :)